src/autoscaler/db/db.go

```diff
@@ -34,3 +34,8 @@ type HistoryDB interface {
 	CanScaleApp(appId string) (bool, error)
 	Close() error
 }
+
+type ScheduleDB interface {
```

Comment:
Just want to make sure we aren't losing things which is why I asked. Separate story is fine.